Utilities/js.cpp

```diff
@@ -30,7 +31,10 @@
 #include <LibTextCodec/Decoder.h>
 #include <signal.h>
 
-RefPtr<JS::VM> g_vm;
+// FIXME: https://github.com/LadybirdBrowser/ladybird/issues/2412
+//    We should be able to destroy the VM on process exit.
```

Comment:
This FIXME doesn't seem like something we should waste energy pursuing.
What value is added by destroying the VM on process exit, other than making it take longer and forcing us to deal with bugs that otherwise don't exist?

Reply:
If this is just for debug builds to find leaks etc, that's fine ofc.